### PR TITLE
docs(config): Notify that the "coverage" preprocessor uses a deprecat…

### LINKED ISSUE #3204: Karma Documentation relates to an old version of Istanbul
### Expected behaviour
Karma's documentation should describe how to collect code coverage using the new version of IstanbulJS using the new "karma-coverage-istanbul-reporter" package (https://www.npmjs.com/package/karma-coverage-istanbul-reporter).

### Actual behaviour
Karma's documentation should describes how to collect code coverage using the old version of Istanbul  (https://github.com/gotwarlost/istanbul) which is deprecated two years ago and doesn't support new versions. Currently this is being done by using the "karma-coverage" package (https://www.npmjs.com/package/karma-coverage).

### Steps to reproduce the behaviour
Checkout this link: https://karma-runner.github.io/3.0/config/preprocessors.html

### PR DESCRIPTION
**Pull request by nadavye**
…ed version of the coverage tool (Istanbul) and add link to a package with the new version (IstanbulJS)

The current config uses "karma-config" which uses a two years old deprecated version of Istanbul.
This may lead to incorrect coverage and other bugs.
Since there are many ways to configure code coverage, but most of them uses the old Istanbul, this causes confusion.
In order to reduce the confusion, a notice has been added and link to the package which comes in Angular. This package uses the the new & maintained version of IstanbulJS (vs. Istanbul, which is deprecated).

Closes #3204

Check that your description matches the automatic change-log format:
http://karma-runner.github.io/latest/dev/git-commit-msg.html
then delete this reminder.

### CODE REVIEW

**Comment by nadavye** [comment]
I have signed

**Comment by johnjbarton** [comment]
Send a PR to fix the problem.

File: docs/config/04-preprocessors.md [hunk 1]
```diff
@@ -20,7 +20,9 @@ Note: Most of the preprocessors need to be loaded as [plugins].
 - [html2js]
   - Note any .html files listed in the files section must be referenced at run time as `window.__html__['template.html']`. [Learn more](https://github.com/karma-runner/karma-html2js-preprocessor#how-does-it-work-).
   - If this preprocessor is disabled, included .html files will need `base/` added to beginning of their path reference. See [discussion in issue 788][issue788].
-- [coverage]
+- [coverage] 
+  - Note this preprocessor uses a deprecated version of [Istanbul](https://github.com/gotwarlost/istanbul) which may lead to unaccurate code coverage. It is advised to use [karma-coverage-istanbul-reporter](https://www.npmjs.com/package/karma-coverage-istanbul-reporter) as it uses [IstanbulJS](https://github.com/istanbuljs/istanbuljs) which is the new and maintained version.
```
Comment (johnjbarton) [inline docs/config/04-preprocessors.md]: Just describe the karma-coverage-istanbul-reporter and its positive characteristics.

**Comment by johnjbarton** [comment]
The negative tone of the description caused me to close it without reading the content, sorry.

**Pull request closed without merging**